### PR TITLE
Progress information

### PR DESCRIPTION
**Pull request by AronBuzogany**
This PR fixes [issue #11](https://github.com/SolidLabResearch/generic-data-viewer/issues/11) as follows:

- When the timer stops, the query is done executing
- Additional info is provided by a counter that counts the results

### CODE REVIEW

File: src/components/Time.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (pheyvaer) [inline src/components/Time.jsx:1]: Remove empty lines.

**Review by pheyvaer — CHANGES_REQUESTED** [review]

File: src/components/Time.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+function Time(props){
+    const time = props.time 
+    const minutes = Math.floor((time % 360000) / 6000) + 60*Math.floor(time / 360000)
+    const seconds = Math.floor((time % 6000) / 100) + 60*minutes
+    const milliseconds = time % 100 
+
+    return(
+        <>
+        {seconds}.{milliseconds}s
```
Comment (pheyvaer) [inline src/components/Time.jsx:13]: Only keep the seconds. There is not need for the milliseconds. Or make it configurable.

**Comment by pheyvaer** [comment]
The results/duration flashes to the left and right. That should not happen. Also it flashes when there are not results yet.

File: src/components/Time.jsx (hunk 1 above)
Comment (AronBuzogany) [reply src/components/Time.jsx]: I have removed those lines in [commit 0c7598d](https://github.com/SolidLabResearch/generic-data-viewer/pull/17/commits/0c7598de586674e3f7ec4a020a19d1115c1df3cf)

File: src/components/Time.jsx (hunk 2 above)
Comment (AronBuzogany) [reply src/components/Time.jsx]: I have made it configurable in [commit 4cce533](https://github.com/SolidLabResearch/generic-data-viewer/pull/17/commits/4cce533732da341c200f375b6f998969a68ac15d) and updated the README respectively in [commit 21c9821](https://github.com/SolidLabResearch/generic-data-viewer/pull/17/commits/21c9821a5c36f6a88d55113293486671daa2a390)

**Comment by AronBuzogany** [comment]
> The results/duration flashes to the left and right. That should not happen. Also it flashes when there are not results yet.

This has been resolved in PR #23

File: src/right-field/RightField.jsx [hunk 3]
```diff
@@ -1,26 +1,183 @@
...
+              {results.length}
+            </label>
+          )}
+          {selectedQuery && (
+            <strong id="query-name-label">{selectedQuery.name}</strong>
+          )}
+          {selectedQuery && (
+            <label className="stopWatch">
+              <strong>Runtime:</strong>
+              <Time time={time} showMilliseconds={config.showMilliseconds} />
```
Comment (pheyvaer) [inline src/right-field/RightField.jsx:89]: Once the query is actually done, it should also explicitly mention that.

**Review by pheyvaer — CHANGES_REQUESTED** [review]

File: src/right-field/RightField.jsx (hunk 3 above)
Comment (AronBuzogany) [reply src/right-field/RightField.jsx:90]: I have done this by replacing "Runtime: " with "Finished in:" in [commit 7e15f24](https://github.com/SolidLabResearch/generic-data-viewer/pull/17/commits/7e15f24a8eb9cbc311fb3d8b90e26aa07c142398). Is this change obvious and user friendly?

Comment (pheyvaer) [reply src/right-field/RightField.jsx:90]: That's good!

**Review by pheyvaer — APPROVED** [review]

**Pull request merged**